docs/CaliptraIntegrationSpecification.md

```diff
@@ -725,6 +725,145 @@ The following code snippet and schematic diagram illustrate JTAG originating CDC
+
+1. Caliptra Core IP assumes the primary reset inputs are already synchronized to their respective clocks. The integrator should add external reset synchronizers to achieve the same.
+    - *cptra_pwrgood* and *cptra_rst_b* resets should be synchronzied to *cptra_clk*
+    - *jtag_trst_n* reset should be synchronzied to *jtag_clk*
+2. The following debug registers which are driven from JTAG will not be toggled during functional flow.
+    - u_caliptra.rvtop.veer.dbg.dmcontrol_reg[0] = 0
+3. Set *scan_mode* to 0 for functional analysis.
+4. Stamp/Create functional resets for *cptra_noncore_rst_b* and *cptra_uc_rst_b* at the definition points as mentioned in *Table 20*. 
+5. Create funtional reset grouping - This step need to be customized as per the EDA tool which is being used for RDC analysis. Basically the thought process here is to achieve the below three sequencing requirements/constraints:-
+    - Gate all clocks when *cptra_noncore_rst_b* is asserted. This will ensure that the capture flop clock is gated while the source flop's reset is getting asserted, therby preventing it from going metastable. So when *cptra_noncore_rst_b* is going to get asserted, below signals are constrained to be at 1 around that time.
+        - soc_ifc_top1.i_soc_ifc_boot_fsm.rdc_clk_dis
+        - soc_ifc_top1.i_soc_ifc_boot_fsm.arc_IDLE
+    - Gate Veer core clock when *cptra_uc_rst_b* is asserted. This will ensure that the capture flop clock is gated while the source flop's reset is getting asserted, therby preventing it from going metastable. So when *cptra_uc_rst_b* is going to get asserted, below signal is constrained to be at 1 around that time.
+        - soc_ifc_top1.i_soc_ifc_boot_fsm.fw_update_rst_window
+    - Quiesce AHB bus before *cptra_uc_rst_b* is asserted. Since firmware reset request is triggered by SW, it can ensure that all AHB transcations are flushed out before initiating a reset request. So when *cptra_uc_rst_b* is going to get asserted, below signal is constrained to be at 0 around that time.
+        - doe.doe_inst.i_doe_reg.s_cpuif_req
+        - ecc_top1.ecc_reg1.s_cpuif_req
+        - hmac.hmac_inst.i_hmac_reg.s_cpuif_req
+        - key_vault1.kv_reg1.s_cpuif_req
+        - pcr_vault1.pv_reg1.s_cpuif_req
+        - data_vault1.dv_reg1.s_cpuif_req
+        - sha512.sha512_inst.i_sha512_reg.s_cpuif_req
+        - soc_ifc_top1.i_ahb_slv_sif_soc_ifc.dv
+        - sha256.sha256_inst.i_sha256_reg.s_cpuif_req
+        - csrng.u_reg.u_ahb_slv_sif.dv
+        - entropy_src.u_reg.u_ahb_slv_sif.dv
+6. Constraint the RDC false paths as per *Table 21*.
+
+
+## RDC Violations and Waivers
+
+Considering the given constraints, we've identified three sets of crossings that are flagged as RDC violations. All of them can be waived as explained in *Table 22*. It's worth noting that the report may differ across EDA tools due to variations in structural analysis, which can be influenced by a range of settings.
```

Comment:
I think we're generally trying to avoid 1st person pronouns in the spec - can this be reworded as generic implementation point of view instead of "we/our"?

Reply:
reworded, please check

Reply:
Looks great, thanks